Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,104 @@
+
+[color=#ff0000]Security Rules[/color]
+
+[color=#a4885c]12.[/color] Security is expected to use non-lethal measures unless faced with lethal force or a situation where non-lethal measures aren’t possible.
+    - If you still manage to kill someone as security, get them cloned immediately, and make sure their stuff is returned. Security should make their best effort to keep criminals alive and in custody. The time it takes to have them cloned counts towards serving their sentence.
+[color=#a4885c]13.[/color] Prisoners should not be given sentences longer than 10 minutes. Certain extreme exceptions like bombers and assassins may warrant permabrigging, or in the very worst cases, execution.
+    - Note that employment at a hostile corporation (i.e. traitor status) alone does not qualify someone for perma or execution.
+    - Security are required to follow Space Law. If in doubt, consult Centcom’s law team.
+    - A slightly-too-long sentence for the crime committed is considered an IC issue. Use common sense when considering if a sentence should be brought to admins.
+
+[color=#ff0000]MRP Specific Rules[/color]
+
+[color=#a4885c]14.[/color] This server is a "Roleplay Expected" server. We define this as performing your assigned duties, "doing your job". This means it is important to do what is expected out of your department and not what would be expected out of other departments.
+    - Don’t do other peoples jobs for them. Opt in for the role you intend to play or change your job by visiting the Head of Personnel. Failure to do your basic duties may result in a job ban.
+
+[color=#a4885c]15.[/color] Each round exists independently of all other rounds. While friendships persisting through rounds is okay, using experiences from other rounds to give an advantage or disadvantage to other players is not permitted.
+
+[color=#a4885c]16.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living. Even if you are cloned, you do not remember the events culminating in your death.
+
+[color=#a4885c]17.[/color] Do your best not to interact negatively with SSD/AFK players. Moving them to safety is acceptable; killing, looting, or otherwise griefing them while they are away is not.
+    - SSD characters are players who are disconnected or AFK. It is possible for players to return to SSD bodies. Players become catatonic when they take a ghost spawner role or commit suicide. Players are NOT able to return to these bodies without admin intervention.
+    - If the player in question is an antag’s target or they are being secured by security, interactions to finish what is required of your duties/objectives are always acceptable.
+    - Ahelping about an SSD/AFK player may grant an exception to this rule.
+    - This does not apply to players that are catatonic, although needlessly killing or harming catatonic players is discouraged.
```

Comment:
some version of this should be added to the original ruleset.

Reply:
also agree